udf-compiler/src/main/scala/com/nvidia/spark/udf/Instruction.scala

```diff
@@ -106,6 +106,8 @@ private object Repr {
       LocalDateTime(new ParseToTimestamp(text, formatter.pattern))
     }
   }
+
+  case class ClassTag[T](classTag: scala.reflect.ClassTag[T]) extends CompilerInternal("scala.reflect.ClassTag")
```

Comment:
could this have a name that doesn't conflict with the scala `ClassTag`?

Reply:
It isn't different from other classes in `Repr` such as `Repr.StringBuilder`.  It is used as `Repr.ClassTag`, which is different than the scala `ClassTag`.

Reply:
I think if we add code that is called exactly the same as other code, just with a package difference (especially core scala language code) that's bound to confuse someone looking at this.

Reply:
@seanprime7 thought about this one some more. You use `Repr.` consistently before these which is enough IMHO. I do not think one could import `Repr._` and then get these to look just like the built-in ones, but in either case that's not happening here.